frontend/src/app/[locale]/process/ProcessMilestones.tsx

```diff
@@ -38,90 +40,97 @@ const ProcessMilestones = () => {
         data-test-id="process-high-level-content"
         titleSize="m"
         bottomBorder="dark"
         gridGap={6}
       >
-        {keys.map((key, index) => {
-          const title = t(`milestones.icon_list.${key}.title`);
-          const content = t.rich(`milestones.icon_list.${key}.content`, {
-            p: (chunks) => (
-              <p className="font-sans-md line-height-sans-4 desktop-lg:line-height-sans-6">
-                {chunks}
-              </p>
-            ),
-            italics: (chunks) => <em>{chunks}</em>,
-          });
+        <IconList className="usa-icon-list--size-lg grid-row">
+          {high_level_roadmap_items.map((_unusedItem, index) => {
+            const title = t(`high_level_roadmap_items.${index}.title`);
+            const content = t.rich(
+              `high_level_roadmap_items.${index}.content`,
+              {
+                p: (chunks) => (
+                  <p className="font-sans-md line-height-sans-4 desktop-lg:line-height-sans-6">
+                    {chunks}
+                  </p>
+                ),
+                italics: (chunks) => <em>{chunks}</em>,
+              },
+            );
 
-          return (
-            <Grid key={title + "-key"} tabletLg={{ col: 4 }}>
-              <IconList className="usa-icon-list--size-lg">
-                <IconListItem className="margin-top-4">
-                  <IconListIcon>{getIcon(index)}</IconListIcon>
-                  <IconListContent className="tablet-lg:padding-right-3 desktop-lg:padding-right-105">
-                    <IconListTitle className="margin-bottom-2" type="h3">
-                      {title}
-                    </IconListTitle>
-                    <div className="font-sans-md line-height-sans-4 desktop-lg:line-height-sans-6">
-                      {content}
-                    </div>
-                    {
-                      // Don't show the chevron in the last row item.
-                      index < keys.length - 1 ? (
-                        <USWDSIcon
-                          className="usa-icon usa-icon--size-9 display-none tablet-lg:display-block text-base-lighter position-absolute right-0 top-0 margin-right-neg-5"
-                          name="navigate_next"
-                        />
-                      ) : (
-                        ""
-                      )
-                    }
-                  </IconListContent>
-                </IconListItem>
-              </IconList>
-            </Grid>
-          );
-        })}
+            return (
+              <IconListItem
+                key={title + "-key"}
+                className="margin-top-4 tablet-lg:grid-col-4"
+              >
+                <IconListIcon>{getIcon(index)}</IconListIcon>
+                <IconListContent className="tablet:padding-right-7">
+                  <h3
+                    aria-label={`Step ${index + 1}: ${title}`}
+                    className="margin-bottom-2 usa-icon-list__title"
```

Comment:
I'm curious why the change from using the IconListTitle component to what presumably is the same thing but in class appended to native html element form? Is this the way the docs have it now?

Reply:
I had to do this in order to add the aria-label as requested. Using the trussworks component you can add classes directly to the h3, but you can't pass anything else through. As far as I could tell, the only thing the component was really doing was adding USWDS classes, which I replicated directly. Does that make sense?